src/components/ProfileForm/profileFormDefs.js

```diff
@@ -95,8 +99,18 @@ CityStructDef.getValidationErrorMessage = (city): string => {
   return 'Please specify your city';
 };
 
+PhoneStructDef.getValidationErrorMessage = (phone): string => {
+  if (phone) {
+    if (!isValidUKPhone(phone)) {
+      return 'Please enter a valid UK phone number with +44';
```

Comment:
Why only UK numbers?

Reply:
Hope this will clarify, here -> https://github.com/pillarwallet/pillarwallet/pull/1085#discussion_r278007537